packages/jest-config/src/normalize.ts

```diff
@@ -1022,6 +1023,8 @@ export default async function normalize(
     newOptions.onlyChanged = newOptions.watch;
   }
 
+  newOptions.randomize = newOptions.randomize || argv.randomize;
+
   newOptions.showSeed = newOptions.showSeed || argv.showSeed;
```

Comment:
```suggestion
  newOptions.showSeed = newOptions.randomize || newOptions.showSeed || argv.showSeed;
```

not showing the seed when running randomized tests doesn't make sense to me